service/controller/resource/namespace/current.go

```diff
@@ -71,7 +74,12 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	// remove the namespace as usual.
 	if key.IsDeleted(&customObject) {
 		n := key.ClusterNamespace(customObject)
-		list, err := r.k8sClient.CoreV1().Pods(n).List(ctx, metav1.ListOptions{})
+
+		lo := metav1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s", label.ManagedBy, project.Name()),
+		}
+
+		list, err := r.k8sClient.CoreV1().Pods(n).List(ctx, lo)
```

Comment:
After this, we don't count app-operator per cluster namespace anymore.

Reply:
I think we set the managed-by label on the deployment but not in the template. Can you check the pods have this label?

Reply:
You're right. It's on the deployment side; 
```
 k get -n 60tzw deploy worker-zsa9a -o yaml | yq read - "metadata.labels"
app: worker
cluster: 60tzw
customer: giantswarm
giantswarm.io/cluster: 60tzw
giantswarm.io/managed-by: kvm-operator
giantswarm.io/organization: giantswarm
node: zsa9a
```

```
 k get po -n 60tzw worker-zsa9a-785788964c-qt5ht -o yaml | yq read - "metadata.labels"
app: worker
cluster: 60tzw
customer: giantswarm
giantswarm.io/cluster: 60tzw
giantswarm.io/organization: giantswarm
kvm-operator.giantswarm.io/pod-watcher: kvm-operator
node: zsa9a
pod-template-hash: 785788964c
```

I will fix this, sorry!